File: lecture0-30/lecture001/largest.java
import java.util.Scanner;
//verified
public class largest{
    public static void main(String[] args){
        Scanner scn=new Scanner(System.in);

        System.out.print("Enter a: ");
        int a=scn.nextInt();

        System.out.print("Enter b: ");
        int b=scn.nextInt();

        System.out.print("Enter c: ");
        int c=scn.nextInt();

        if(a>b && a>c)
        {System.out.print("Largest is a");}

        else if(b>a && b>c)
        {System.out.print("Largest is b");
        }
        else
        {System.out.print("Largest is c");}
    }
}

File: lecture001/largest.cpp
#include <iostream>
using namespace std;
int main()
{
    int a, b, c;
    cout << "Enter 3 numbers : ";
    cin >> a >> b >> c;
    if (a > b && a > c)
    {
        cout << a << " is largest ";
    }
    else if (b > a && b > c)
    {
        cout << b << " is largest ";
    }
    else
    {
        cout << c << " is largest ";
    }
}